Q3_f.sql

```diff
@@ -0,0 +1,46 @@
+-- problem f
+
+-- Print all information (mid, title, year, num ratings, rating) for the movie(s) with the
+-- highest number of ratings.
+
+select * from movies
+where num_rating = (select MAX(num_rating) from movies);
```

Comment:
use num_ratings ?

Reply:
I checked the export_schema and it is num_rating not num_ratings so this should work fine with our tables.